nan_typedarray_contents.h

```diff
@@ -0,0 +1,82 @@
+// Copyright Joyent, Inc. and other Node contributors.
```

Comment:
Did you mean nan_string_bytes.h?  I don't see Joyent copyright boilerplate anywhere else.

Aside, I speculate @indutny included it because it was copied from joyent/node but I think we can just remove it; most of that code was originally written by me.